src/main/java/edu/umich/lib/dor/replicaexperiment/service/Purge.java

```diff
@@ -0,0 +1,63 @@
+package edu.umich.lib.dor.replicaexperiment.service;
+
+import edu.umich.lib.dor.replicaexperiment.domain.InfoPackage;
+import edu.umich.lib.dor.replicaexperiment.domain.Replica;
+import edu.umich.lib.dor.replicaexperiment.domain.Repository;
+import edu.umich.lib.dor.replicaexperiment.exception.NoEntityException;
+
+public class Purge implements Command {
+    private ReplicaService replicaService;
+    private String packageIdentifier;
+
+    private Replica replica;
+    private RepositoryClient repositoryClient;
```

Comment:
As I was working on this, I realized the other `Command`s are a bit of a mess in their use of instance variables. Many of them aren't necessary, and none of them are marked `private`. I'm gonna follow this PR up with a quick one to tidy up, see #21.